backend/corpora/lambdas/api/v1/dataset.py

```diff
@@ -0,0 +1,39 @@
+from flask import make_response, jsonify
+
+from ....common.entities import Dataset
+from ....common.utils.db_utils import db_session
+from ....common.utils.exceptions import NotFoundHTTPException, ServerErrorHTTPException
+
+
+@db_session
+def post_dataset_asset(dataset_uuid: str, asset_uuid: str):
+
+    # retrieve the dataset
+    dataset = Dataset.get(dataset_uuid)
+    if not dataset:
+        raise NotFoundHTTPException(f"'dataset/{dataset_uuid}' not found.")
```

Comment:
Does these exceptions work with the `problem` schema in the API?

Reply:
Yes, connexion defines a problemException that follows that format, and all of the HTTP error inherit it.